### ISSUE TITLE
Please update dependency versions OSV:GHSA-2H63-QP69-FWVW

### ISSUE THREAD

**Issue by LouDnl** [suvash/one-time#15]
According to [Grype](https://github.com/anchore/grype) the dependencies batik-util and batik-xml that one-time uses are vulnerable to Server-side request forgery (SSRF). Please provide an update.

[OSV:GHSA-2H63-QP69-FWVW](https://vulners.com/osv/OSV:GHSA-2H63-QP69-FWVW)

```shell
 ✔ Vulnerability DB        [no update available]
 ✔ Indexed target/package.jar
 ✔ Cataloged packages      [112 packages]
 ✔ Scanned image           [2 vulnerabilities]
NAME        INSTALLED  FIXED-IN  TYPE          VULNERABILITY        SEVERITY
batik-util  1.10       1.14      java-archive  GHSA-2h63-qp69-fwvw  High
batik-xml   1.10       1.14      java-archive  GHSA-2h63-qp69-fwvw  High
```

Dependency tree:
```shell
[one-time "0.7.0" :exclusions [[commons-codec]]]
   [com.github.kenglxn.qrgen/javase "2.6.0"]
     [com.github.kenglxn.qrgen/core "2.6.0"]
     [com.google.zxing/javase "3.3.0"]
       [com.beust/jcommander "1.48"]
       [com.github.jai-imageio/jai-imageio-core "1.3.1"]
       [com.google.zxing/core "3.3.0"]
     [org.apache.xmlgraphics/batik-dom "1.10"]
       [org.apache.xmlgraphics/batik-css "1.10"]
       [org.apache.xmlgraphics/batik-ext "1.10"]
       [org.apache.xmlgraphics/batik-xml "1.10"]
       [xalan "2.7.2"]
         [xalan/serializer "2.7.2"]
       [xml-apis/xml-apis-ext "1.3.04"]
       [xml-apis "1.3.04"]
     [org.apache.xmlgraphics/batik-svggen "1.10"]
       [org.apache.xmlgraphics/batik-awt-util "1.10"]
         [org.apache.xmlgraphics/xmlgraphics-commons "2.2"]
       [org.apache.xmlgraphics/batik-util "1.10"]
         [org.apache.xmlgraphics/batik-constants "1.10"]
         [org.apache.xmlgraphics/batik-i18n "1.10"]
   [ring/ring-codec "1.1.2"]
```

**Comment by suvash** [comment]
Hi @LouDnl, Thanks for bringing this to attention.
Any chances you can make a PR for this ? I'd happily review that, just a bit busy for a couple of weeks otherwise. I'll look into it during my next free time.

**Comment by LouDnl** [comment]
It's been there for a while now, I'll see what I can do.

**Comment by LouDnl** [comment]
PR: https://github.com/suvash/one-time/pull/16

**Comment by LouDnl** [comment]
New PR: #17

**Comment by suvash** [comment]
Closed by #17
